packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaForm.tsx

```diff
@@ -90,8 +93,8 @@ type TableColumnSchemaFormProps = {
   /* If this is an entity view, the ViewScope can be used to determine the default column models and fetch annotation column models */
   viewScope?: ViewScope
   initialData?: ColumnModel[]
-  onSubmit: (formData: ColumnModelFormData[]) => void
-  isSubmitting: boolean
+  onSubmit?: (formData: ColumnModelFormData[]) => void
+  isSubmitting?: boolean
```

Comment:
SWC won't use these, but they will be used if/when we convert the "next layer" to React components.